### PR TITLE
made a button components and reused it in many pages

### PR DESCRIPTION
**Pull request by revengemiroz**

### CODE REVIEW

File: client/src/components/Button/style.js [hunk 1]
```diff
@@ -1,25 +1,117 @@
...
+    max-width: ${({ maxWidth }) => {
+      switch (maxWidth) {
+        case 'big':
+          return '120px';
+        case 'medium':
+          return '95px';
+        case 'small':
+          return '60px';
+        default:
+          return '1000px';
```
Comment (ivanms1) [inline client/src/components/Button/style.js]: width: auto;

File: client/src/components/Button/style.js [hunk 2]
```diff
@@ -1,25 +1,117 @@
...
+          return '10px 0px';
+        case 'approve':
+          return '7px 12px';
+        default:
+          return '10px 0px';
+      }
+    }};
+
+    max-width: ${({ maxWidth }) => {
+      switch (maxWidth) {
```
Comment (ivanms1) [inline client/src/components/Button/style.js]: are you sure you want to hardcode width?
the button will not look good if the text gets too blong

File: client/src/components/Button/style.js (hunk 1 above)
Comment (revengemiroz) [reply client/src/components/Button/style.js]: done

File: client/src/components/Button/style.js (hunk 2 above)
Comment (revengemiroz) [reply client/src/components/Button/style.js]: done

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**